pytensor/tensor/basic.py

```diff
@@ -760,7 +760,31 @@ def switch(cond, ift, iff):
     """if cond then ift else iff"""
 
 
-where = switch
+def where(cond, ift=None, iff=None):
```

Comment:
Perhaps add kwargs that can be passed to `nonzero` or `switch`? We have stuff like `name` that we can add by default to PyTensor variables

```suggestion
def where(cond, ift=None, iff=None, **kwargs):
```